src/main/java/com/internet/shop/dao/jdbc/OrderDaoJdbcImpl.java

```diff
@@ -0,0 +1,185 @@
+package com.internet.shop.dao.jdbc;
+
+import com.internet.shop.dao.OrderDao;
+import com.internet.shop.exceptions.DataProcessingException;
+import com.internet.shop.lib.Dao;
+import com.internet.shop.model.Order;
+import com.internet.shop.model.Product;
+import com.internet.shop.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class OrderDaoJdbcImpl implements OrderDao {
+
+    @Override
+    public Order create(Order order) {
+        try (Connection connection = ConnectionUtil.getConnection(); PreparedStatement statement =
+                connection.prepareStatement("INSERT INTO orders (user_id) VALUES(?)",
+                        Statement.RETURN_GENERATED_KEYS)) {
+            statement.setLong(1, order.getUserId());
+            statement.executeUpdate();
+            ResultSet generatedKeys = statement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                order.setOrderId(generatedKeys.getLong(1));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't create Order with id: "
+                    + order.getOrderId(), e);
+        }
+        for (Product product : order.getProducts()) {
+            addDataToOrdersProductTable(order.getOrderId(), product.getProductId());
+        }
+        return order;
+    }
+
+    @Override
+    public Optional<Order> getById(Long id) {
+        Order order = null;
+        try (Connection connection = ConnectionUtil.getConnection(); PreparedStatement statement =
+                connection.prepareStatement("SELECT order_id, user_id FROM orders "
+                        + "WHERE order_id = ? AND is_deleted = FALSE ")) {
```

Comment:
To make such blocks of code smaller and easier to read, you could do this:
```suggestion
        String query = "SELECT order_id, user_id FROM orders "
                        + "WHERE order_id = ? AND is_deleted = FALSE ";
        try (Connection connection = ConnectionUtil.getConnection(); 
                 PreparedStatement statement =
                          connection.prepareStatement(query)) {
```